src/mod_last.erl

```diff
@@ -244,6 +251,30 @@ remove_domain(Acc, HostType, Domain) ->
     mod_last_backend:remove_domain(HostType, Domain),
     Acc.
 
+-spec user_receive_iq(mongoose_acc:t(), mongoose_c2s:hook_params(), gen_hook:extra()) ->
+    mongoose_c2s_hooks:hook_result().
+user_receive_iq(Acc, _Params, _Extra) ->
+    case mongoose_iq:info(Acc) of
+        {#iq{type = get, xmlns = ?NS_LAST}, Acc1} ->
+            maybe_respond_last(Acc1);
+        {_, Acc1} ->
+            {ok, Acc1}
+    end.
+
+maybe_respond_last(Acc) ->
+    HostType = mongoose_acc:host_type(Acc),
+    {From, To, _} = mongoose_acc:packet(Acc),
+    case {To#jid.lresource, can_respond(HostType, From, To)} of
+        {<<>>, _} ->
+            {ok, Acc};
```

Comment:
Is this clause reachable? Wouldn't it be caught by the IQ handler?

Reply:
Actually, that is true!

Reply:
The XMPP core itself says that IQs to a user without specifying the resource must be handled by the server on their behalf: https://www.rfc-editor.org/rfc/rfc6120#section-10.5.3.2